packages/SketchMorph2-Core.package/M2Backend.class/instance/drawRectangle.st

```diff
@@ -1,6 +1,10 @@
 drawing
 drawRectangle
 	self isErasing: false.
-	self pen sourceForm: ((Form extent: self rectangleBrushSize) fillBlack).
+	self pen sourceForm:
+		(Form
```

Comment:
I'm not sure the style is correct here since the following is not a cascade.

Reply:
How would you suggest to format this snippet without exceeding the character limit per line?

Reply:
Moving the `extent:` line to the same line as `(Form`. I think this was the style for such cases.

Reply:
Like so?

```smalltalk
self pen sourceForm:
	(Form extent: self rectangleBrushSize
		offset: (self penNibOffset: self rectangleBrushSize))
		fillBlack.
```

Reply:
More like the first one. However, `fillBlack` is a method not called on `Form` but rather on the result of `Form class>>extent:offset:`. I'm actually not sure how this should be made. Maybe put `fillBlack.` on the line above, directly behind the `))`?

Reply:
So a mix of the two versions 😅.